cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,42 @@
-describe('', () => {
-  before(() => {
+describe('Article Testing', () => {
+  // Assuming you need to log in before each test
+  beforeEach(() => {
+    // Visit the page
+    cy.visit('https://conduit.mate.academy/');
 
-  });
+    // Click "Sign In"
```

Comment:
You are supposed to use the `login()` function located in the `commands.js` file.